### DISCUSSION TITLE
Ability to export only background image

### DISCUSSION THREAD

**Discussion by belugawhale3** [vinothpandian/react-sketch-canvas#91] [Ideas]
Would like to be able to export only the background image after its scaled to fit the current canvas size. currently you can only export the drawing with or without the background image

**Comment by vinothpandian** [comment]
Thanks for your suggestion! However, I would like to keep ReactSketchCanvas as a simple sketching tool. I recommend [react-konva](https://github.com/konvajs/react-konva) for advanced functionality.
